static/templates/draft_table_body.hbs

```diff
@@ -22,6 +22,9 @@
                     {{#each narrow_drafts}}
                         {{> draft}}
                     {{/each}}
+                    <button type="button" id="compose-new-message-button" class="button sea-green">
```

Comment:
I think we should keep `compose-new-message-button` as class and use `#draft_overlay .compose-new-message-button` to reference it so that I know where this button is placed regardless of the file I am reading.